src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,63 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  filterBy: string,
+  handleStatus: (str: string) => void,
+  query: string,
+  setQuery: (str: string) => void,
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = ({
+  filterBy,
+  handleStatus,
+  query,
+  setQuery,
+}) => {
+  return (
+    <form
+      className="field has-addons"
+      onSubmit={(event) => event.preventDefault()}
+    >
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            className="option"
+            value={filterBy}
+            onChange={(event) => handleStatus(event.target.value)}
```

Comment:
It will be great to add separate methods here and in other onClicks & onChanges)